fanficfare/configurable.py

```diff
@@ -522,7 +522,7 @@ class Configuration(configparser.SafeConfigParser):
 
     def __init__(self, sections, fileform, lightweight=False):
         site = sections[-1] # first section is site DN.
-        configparser.SafeConfigParser.__init__(self)
+        configparser.ConfigParser.__init__(self)
```

Comment:
This leaves a mismatch between the parent class and the called `__init__` method.

Also, is it tested backwards compatible to Python 2.7?  Calibre is still on 2.7.

Reply:
Ok, my bad. I tried to fix the warnings but I didn't understand enough what I was doing...
```

fanficfare/adapters/tests/adapters/test_adapter_chireadscom.py::TestExtractChapterUrlsAndMetadata::test_raise_404_for_unexistant_story
[...]
fanficfare/adapters/tests/adapters/test_adapter_chireadscom.py::TestGetChapterText::test_get_metadata
  ~/src/perso/FanFicFare/fanficfare/configurable.py:525: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
    configparser.SafeConfigParser.__init__(self)
```